cmd/swarmctl/main.go

```diff
@@ -41,7 +41,7 @@ func defaultSocket() string {
 	if swarmSocket != "" {
 		return swarmSocket
 	}
-	return "./swarmkitstate/swarmd.sock"
+	return "/run/swarmd/swarmd.sock"
```

Comment:
On closer reading you are right that `/var/run/swarmd.sock` makes more sense than either `/var/run/swarmd/swarmd.sock` (which I mistakenly read as your suggestion) or `/run/swarmd/swarmd.sock`. I'll make that change too.